test/bin/Rules.mk

```diff
@@ -30,18 +30,18 @@ TGTS_GX_$(d) := hang-fds
 TGTS_GX_$(d) := $(addprefix $(d)/,$(TGTS_GX_$(d)))
 
 $(TGTS_GX_$(d)):
-	go build -i $(go-flags-with-tags) -o "$@" "$(call gx-path,$(notdir $@))"
```

Comment:
Do you remember why we needed to remove this flag?

Reply:
IIRC when cross compiling, go would try to write objects to /usr

Reply:
Ah. Yeah, that makes sense.